packages/documentation/docs/for-developers/for-blueprint-developers/HOLD.md

```diff
@@ -0,0 +1,40 @@
+# HOLD
+
+HOLD is a feature in Sofie to allow for a special form of take between two parts. It allows for the new part to start with some portions of the old part being retained, with the next 'take' stopping the remaining portions of the old part and not performing a true take.
+
+For example, it could be setup to hold back the video when going between two clips. The first take would start the audio from the second clip, but keep the video from the first clip. The second take would stop the first clip, and show the video for the second clip.
+
+## Flow
+
+While HOLD is active or in progress, an indicator is shown in the header of the UI.  
+![HOLD in Rundown View header](/img/docs/rundown-header-hold.png)
+
+It is not possible to run any adlibs while a hold is active, or to change the nexted part. Once it is in progress, it is not possible to abort or cancel the HOLD and it must be run to completion. If the second part has an autonext and that gets reached before the HOLD is completed, the HOLD will be treated as completed and the autonext will execute as normal.
+
+When the part to be held is playing, with the correct part as next, the flow for the users is:
+
+- Activate HOLD (By hotkey or other source)
+- Perform a take into the HOLD
+- Perform a take to complete the HOLD
+
+Before the first take in the HOLD, it can be cancelled in the same way it was activated.
```

Comment:
I think a sentence describing what is happening to the parts, i.e. (which one is next, which one is current and when etc.) would be good here.